image/image.go

```diff
@@ -120,7 +120,7 @@ func unpack(w walker, dest, refName string) error {
 		return err
 	}
 
-	if err = ref.validate(w, validRefMediaTypes); err != nil {
+	if err = ref.validate(w, []string{v1.MediaTypeImageManifest}); err != nil {
```

Comment:
I don't think so. If the `ref` of an OCI image  refer to a manifest list, and the the manifest list refer to a manifest,  the `findManifest` below are supposed to get the `manifest` from the `manifest list`. With your changes, this process are break.